scanner/updater/export.go

```diff
@@ -217,3 +219,26 @@ func (t *rateLimitedTransport) RoundTrip(req *http.Request) (*http.Response, err
 	}
 	return t.transport.RoundTrip(req)
 }
+
+func epssOpts(ctx context.Context) []updates.ManagerOption {
+	enricher := &epss.Enricher{}
+	configJSON, err := json.Marshal(epss.Config{
+		URL: nil, // use default URL for now
+	})
+	if err != nil {
+		log.Printf("Failed to encode EPSS enricher config : %v", err)
+		return nil
+	}
+
+	err = enricher.Configure(ctx, func(config interface{}) error {
```

Comment:
No need to call this, as it'll be called by `mgr.Run(ctx)` in the `bundle` function above. You should set the configuration the same way it's done for `nvd` in `nvdOpts`